src/renderer/cpu.rs

```diff
@@ -42,9 +42,25 @@ fn cpu_title<'a>(app: &'a CPUTimeApp, histogram: &'a [u64]) -> Line<'a> {
             .map(|s| format!("{: >3.0}", s.current_temp))
             .collect::<Vec<String>>()
             .join(",");
-        format!(" TEMP [{t:}°C]")
+
+        let hot_threshold = 70_f64;
```

Comment:
Curious how these thresholds were determined. I imagine that there is also some processor variety in acceptable temperatures ?